### PR TITLE
Refacto gruntfile

### PR DESCRIPTION
**Pull request by itkg-ngilain**
https://github.com/open-orchestra/open-orchestra/pull/492
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/799

### CODE REVIEW

**Comment by nicolasThal** [comment]
:+1: après un petit rebase qui devrait corriger le build

File: GruntTasks/Options/concat.orchestrajs.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+          'web/built/openorchestrabackoffice/js/table/TableviewAction.js',
+          'web/built/openorchestrabackoffice/js/table/TableviewCollectionView.js',
+          'web/built/openorchestrabackoffice/js/table/tableviewLoader.js',
+          // MEDIATHEQUE
+          'web/built/openorchestramediaadmin/js/mediatheque/*.js',
+          // USER
+          'web/built/openorchestrauseradmin/js/user/*.js',
+          // INDEXATION
+          'web/bundles/openorchestraindexation/js/*.js',
+
```
Comment (itkg-nanne) [inline GruntTasks/Options/concat.orchestrajs.js]: delete return?

**Comment by itkg-nanne** [comment]
Maybe each comment in files should be preceded with a return?
For me it's :+1:

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**